File: 3221-find-the-peaks/3221-find-the-peaks.java
class Solution {
    public List<Integer> findPeaks(int[] mountain) {
        ArrayList<Integer> res = new ArrayList<Integer>();
        int l = 1;
        int r = 2;
        int n = mountain.length;
        while(l<n && r <n){
            if(mountain[l]>mountain[r] && mountain[l]> mountain[l-1]){
                res.add(l);
                l++;
                r++;
            }
            else{
                l++;
                r++;
            }

        }
        return res;
        
    }
}

File: 3221-find-the-peaks/3221-find-the-peaks.py
class Solution:
    def findPeaks(self, mountain: List[int]) -> List[int]:
        res = []
        n = len(mountain)
        l,r = 1,2
        while l < n and r < n:
            if mountain[l] > mountain[r] and mountain[l]  > mountain[l-1]:
                res.append(l)
            l += 1
            r += 1
        return res
